lib/internal/webstreams/util.js

```diff
@@ -142,6 +142,8 @@ function transferArrayBuffer(buffer) {
 function isDetachedBuffer(buffer) {
   if (ArrayBufferGetByteLength(buffer) === 0) {
     // TODO(daeyeon): Consider using C++ builtin to improve performance.
+    // Replace this when current v8 is updated to include the following commit:
+    // https://chromium.googlesource.com/v8/v8/+/9df5ef70ff18977b157028fc55ced5af4bcee535
```

Comment:
This commit seems trivial, why don't we just backport it now?

Reply:
Yes, that's better. I've opened https://github.com/nodejs/node/pull/45474, and closing this pull request.